### PR TITLE
Removed All Possible Bloats

### PR DESCRIPTION
**Pull request by loannaflip**
In this commit I've tried to remove all bloats as much as possible. Also I've rearranged pretty much everything in a good order. Overall code is more standard and high quality than before.

### CODE REVIEW

**Comment by loannaflip** [comment]
@JesterOrNot @Kreyren Ping!!!

**Comment by Kreyren** [comment]
@Uniminin Did you make any benchmarks for this?

**Comment by Kreyren** [comment]
lgtm excluding results from benches

File: gui/gui.py [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
+
+
+def main():
+    layout = [
+        [sg.Text('Welcome To DockerFreeze GUI')],
+        [sg.T(" " * 7), sg.Text('Gitpod Ready', size=(12, 1)), sg.Checkbox("", default=False)],
+        [sg.T(" " * 3), sg.Submit("BUILD"), sg.Cancel()]
+    ]
+
+    mainWindow = sg.Window('Docker Freeze Gui', default_element_size=(40, 1)).Layout(layout)
```
Comment (JesterOrNot) [inline gui/gui.py]: 2 things,
1. Dockerfreeze is all one word
2. GUI is all caps

**Review by JesterOrNot — CHANGES_REQUESTED** [review]
Thank you! :tada:
Just a few suggestions

**Comment by loannaflip** [comment]
@JesterOrNot There you go. You're most welcome.

**Comment by JesterOrNot** [comment]
LGTM

**Pull request merged**